app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_parts = command.split()
+
+    source = command_parts[1]
+    destination = command_parts[2]
```

Comment:
You should additionally check that the command itself is `mv`, only in that case logic makes sense.